omdb-client/impl/src/main/java/com/garret/movies/omdb/impl/config/OmdbConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.garret.movies.omdb.impl.config;
+
+import com.omertron.omdbapi.OmdbApi;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OmdbConfig {
+
+    @Value("${api_key}")
+    private String API_KEY;
+
+    @Bean
+    public OmdbApi omdbApi() {
```

Comment:
@Garret-vin You can pass API key as argument to this method:

` public OmdbApi omdbApi(@Value("${api_key}") String key) `